src/test/run-pass/issue-60989.rs

```diff
@@ -0,0 +1,4 @@
+fn main() {
+    let c1 = ();
+    c1::<()>;
```

Comment:
Wait how would this compile successfully?!

Reply:
oops
it would not
should have been a compile-fail test